Wikipedia/Code/WMFContinueReadingSectionController.m

```diff
@@ -27,13 +27,12 @@ @interface WMFContinueReadingSectionController ()
 @end
 
 @implementation WMFContinueReadingSectionController
-@synthesize delegate = _delegate;
 
 - (instancetype)initWithArticleTitle:(MWKTitle*)title
                            dataStore:(MWKDataStore*)dataStore {
     NSParameterAssert(title);
     NSParameterAssert(dataStore);
-    self = [super init];
+    self = [super initWithItems:@[self.title]];
```

Comment:
I think this is causing the continue reading crash. Should be `@[title]` not `@[self.title]`.